tools/ci_connector_ops/ci_connector_ops/utils.py

```diff
@@ -138,7 +138,15 @@ def definition(self) -> Optional[dict]:
 
     @property
     def release_stage(self) -> Optional[str]:
-        return self.definition["releaseStage"] if self.definition else None
+        return self.definition["releaseStage"] if self.definition and self.definition.get('releaseStage') else None
+
+    @property
+    def allowed_hosts(self) -> Optional[List[str]]:
+        return self.definition["allowedHosts"] if self.definition and self.definition.get('allowedHosts') else None
+
+    @property
+    def suggested_streams(self) -> Optional[List[str]]:
+        return self.definition["suggestedStreams"] if self.definition and self.definition.get('suggestedStreams') else None
```

Comment:
Nit: I believe this could be simpler
```suggestion
        return self.definition.get('suggestedStreams') if self.definition else None
```